src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/ReadOnlyMemoryConverter.cs

```diff
@@ -2,12 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 
 namespace System.Text.Json.Serialization.Converters
 {
     internal sealed class ReadOnlyMemoryConverter<T> : JsonCollectionConverter<ReadOnlyMemory<T>, T>
     {
         internal override bool CanHaveMetadata => false;
+        public override bool HandleNull => true;
+
+        internal override bool OnTryRead(
```

Comment:
Why do you need to override both?

Reply:
`HandleNull` controls whether the serialization infrastructure should call this converter on `JsonTokenType.Null` values (by default it does that handling automatically), so we need to override it in order to run the custom logic in the `OnTryRead` method.